addons/mail/static/src/models/thread_view.js

```diff
@@ -339,6 +339,11 @@ registerModel({
         },
     },
     fields: {
+        channelMemberListView: one('ChannelMemberListView', {
+            default: insertAndReplace(),
```

Comment:
view should only be computed according to condition on template to display it -> condition on template should only be the existence of the view